lib/deaf_grandma.rb

```diff
@@ -17,11 +17,21 @@ def run!
     end
   end
 
-
+  
   def speak(input)
-
     #Implement your code here <<<<<<<<<
-
+    if input == input.upcase
```

Comment:
rubocop and @paradime recommend using a 'guard clause' instead of wrapping an entire method in an if-statement.